model/src/form/form-definition/types.ts

```diff
@@ -18,7 +18,7 @@ export interface Page {
   path: string
   controller?: string
   components?: ComponentDef[]
-  section: string // the section ID
+  section?: string // the section ID
```

Comment:
Have you used `npm link` to test the local `@defra/forms-model`?

```console
cd forms-designer
npm link --workspace model

cd ../forms-runner
npm link ../forms-designer/model

cd ../forms-manager
npm link ../forms-designer/model
```

I've merged all the ESLint type changes from https://github.com/DEFRA/forms-runner/pull/77 in **forms-runner**

So we can see if ESLint spots usage of `.section` as a now-optional string

Reply:
Yes I have been using `npm link` in the three repos. Previously I had been hacking the `package.json` and pointing the dependency to a local/relative path :face_exhaling: